UnitTests/ScientistTests.cs

```diff
@@ -138,8 +138,8 @@ public void AnExceptionReportsDuration()
             Assert.Equal(42, result);
             mock.Received().Control();
             mock.Received().Candidate();
-            Result<int> observedResult = TestHelper.Results<int>().First(m => m.ExperimentName == "success");
-            Assert.True(observedResult.Matched);
+            Result<int> observedResult = TestHelper.Results<int>().First(m => m.ExperimentName == "failure");
```

Comment:
Yeah I think it would be better.  I was following what was in place to get the test working as expected, but would be happy to change them.